docker/readmodel-db/eservice-template.sql

```diff
@@ -8,11 +8,13 @@ CREATE TABLE IF NOT EXISTS readmodel_eservice_template.eservice_template (
   intended_target VARCHAR NOT NULL,
   description VARCHAR NOT NULL,
   technology VARCHAR NOT NULL,
-  created_at TIMESTAMP WITH TIME ZONE NOT NULL,
-  mode VARCHAR NOT NULL,
-  is_signal_hub_enabled BOOLEAN,
-  PRIMARY KEY (id),
-  CONSTRAINT eservice_template_id_metadata_version_unique UNIQUE (id, metadata_version)
+  created_at TIMESTAMP
+  WITH
+    TIME ZONE NOT NULL,
+    mode VARCHAR NOT NULL,
+    is_signal_hub_enabled BOOLEAN,
+    PRIMARY KEY (id),
+    CONSTRAINT eservice_template_id_metadata_version_unique UNIQUE (id, metadata_version)
```

Comment:
the format is a bit confusing, may I suggest inlining? (same for others)
```suggestion
  created_at TIMESTAMP WITH TIME ZONE NOT NULL,
  mode VARCHAR NOT NULL,
  is_signal_hub_enabled BOOLEAN,
  PRIMARY KEY (id),
  CONSTRAINT eservice_template_id_metadata_version_unique UNIQUE (id, metadata_version)
```

Reply:
Done. It's because we're using a different formatter for some reason. 
There's no global formatter in the monorepo for SQL files.